packages/@okta/i18n/package.json

```diff
@@ -1,83 +1,109 @@
+    "escope": "3.6.0",
+    "espree": "3.5.0",
+    "esprima": "2.7.3",
+    "estraverse": "4.2.0",
+    "esutils": "2.0.2",
     "eventemitter2": "0.4.14",
     "execa": "0.7.0",
     "exit": "0.1.2",
     "file-sync-cmp": "0.1.1",
+    "flat-cache": "1.2.2",
     "fs.realpath": "1.0.0",
     "get-caller-file": "1.0.2",
     "get-stdin": "4.0.1",
     "getobject": "0.1.0",
-    "graceful-fs": "4.1.3",
+    "globals": "9.18.0",
+    "graceful-fs": "4.1.11",
     "grunt-known-options": "1.1.0",
     "hooker": "0.2.3",
+    "ignore": "3.3.3",
+    "imurmurhash": "0.1.4",
     "inflight": "1.0.6",
     "inherits": "2.0.3",
+    "inquirer": "0.12.0",
+    "is-fullwidth-code-point": "1.0.0",
+    "is-my-json-valid": "2.16.0",
+    "is-resolvable": "1.0.0",
+    "isarray": "1.0.0",
     "isexe": "2.0.0",
+    "json-stable-stringify": "1.0.1",
     "lcid": "1.0.0",
+    "levn": "0.3.0",
     "locate-path": "2.0.0",
     "mem": "1.1.0",
     "meow": "3.7.0",
-    "minimatch": "3.0.3",
-    "normalize-package-data": "2.3.5",
+    "minimatch": "3.0.4",
+    "mkdirp": "0.5.1",
+    "normalize-package-data": "2.4.0",
+    "object-assign": "4.1.1",
     "once": "1.4.0",
+    "optionator": "0.8.2",
     "parse-json": "2.2.0",
     "path-is-absolute": "1.0.1",
+    "path-is-inside": "1.0.2",
     "pify": "2.3.0",
+    "pluralize": "1.2.1",
+    "progress": "1.1.8",
     "readable-stream": "2.0.6",
     "require-directory": "2.1.1",
     "require-main-filename": "1.0.1",
+    "require-uncached": "1.0.3",
+    "rimraf": "2.2.8",
+    "safer-buffer": "2.1.2",
     "set-blocking": "2.0.0",
     "sprintf-js": "1.0.3",
     "strip-ansi": "3.0.1",
+    "strip-bom": "3.0.0",
+    "strip-json-comments": "1.0.4",
+    "table": "3.8.3",
+    "text-table": "0.2.0",
+    "user-home": "2.0.0",
+    "util-deprecate": "1.0.2",
+    "wrap-ansi": "2.1.0",
     "y18n": "3.2.1"
   },
   "_requested": {
+    "type": "version",
+    "registry": true,
+    "raw": "@okta/i18n@44.2.0-beta.1",
     "name": "@okta/i18n",
-    "raw": "@okta/i18n@42.3.0",
-    "rawSpec": "42.3.0",
+    "escapedName": "@okta%2fi18n",
     "scope": "@okta",
-    "spec": "42.3.0",
-    "type": "version"
+    "rawSpec": "44.2.0-beta.1",
+    "saveSpec": null,
+    "fetchSpec": "44.2.0-beta.1"
   },
   "_requiredBy": [
     "/"
   ],
-  "_resolved": "https://artifacts.aue1d.saasure.com:443/artifactory/api/npm/npm-okta/@okta/i18n/-/@okta/i18n-42.3.0.tgz",
-  "_shasum": "faab375914e4e5dfc19db22dcb08606a2c45b1e0",
-  "_shrinkwrap": null,
-  "_spec": "@okta/i18n@42.3.0",
-  "_where": "/Users/santhosh/repos/okta/santhoshbalakrishnan/okta-signin-widget",
+  "_resolved": "https://artifacts.aue1d.saasure.com:443/artifactory/api/npm/npm-okta/@okta/i18n/-/@okta/i18n-44.2.0-beta.1.tgz",
+  "_shasum": "c73866b741dd5734a3a224d77f56a03ddd89c669",
+  "_spec": "@okta/i18n@44.2.0-beta.1",
+  "_where": "/Users/astadnikov/okta/okta-signin-widget",
   "bin": {
     "i18n": "dist/bin/i18n.js"
   },
+  "bundleDependencies": false,
```

Comment:
what are these new values? Do you happen to know by any chance?
Taling about `bundleDependencies` and `deprecated`

Reply:
They were added by npm, but seems are optional.
Removed them.